examples/server_readspeed.c

```diff
@@ -13,11 +13,20 @@
 #define UA_SERVER_H_
 #define UA_CONNECTION_H_
 #define UA_TYPES_GENERATED_H_
+#define UA_LOG_H_
+#define UA_SECURITYCONTEXT_H_
+#define UA_SECURITYPOLICY_H_
 #include "server/ua_services.h"
 #include "ua_types_encoding_binary.h"
 
 int main(int argc, char** argv) {
     UA_ServerConfig config = UA_ServerConfig_standard;
+
+    // Set desired security policies
+    UA_SecurityPolicy securityPolicies[] = { UA_SecurityPolicy_None };
```

Comment:
It would probably make sense to add SecurityPolicy#None to the standard config, since it is required for communication to work. The initialization still needs to happen in the server by calling the function. It would maybe be possible to make SecurityPolicy#None not require a call to the init function to remove clutter, but it wouldn't be consistent. What's your thought on that?